src/GT/DB/Prelude.hs

```diff
@@ -0,0 +1,31 @@
+-- | This module should be imported in place of the @GT.Prelude@ module.
+-- Generally speaking, whenever there exists an @.Prelude@ module in the
+-- codebase, prefer to import that instead of the Prelude.
+module GT.DB.Prelude
+    ( mkModel
+    , loadModel
+    -- * Reexports
+    , module GT.Prelude
+    , SqlPersistT
+    , Entity(..)
+    ) where
```

Comment:
I'm a fan of very low friction development - having custom Preludes like this make for an easy place to look for common terms, and it also can help to codify common practices.

It's also nice to just import the big module and then write code rather than messing about with imports.